ext/search/CRM/Search/Upgrader.php

```diff
@@ -41,4 +45,42 @@ public function upgrade_1000() {
     return TRUE;
   }
 
+  /**
+   * Upgrade 1001 - normalize search display column keys
+   * @return bool
+   */
+  public function upgrade_1001() {
+    $this->ctx->log->info('Applying update 1001 - normalize search display columns.');
+    $savedSearches = \Civi\Api4\SavedSearch::get(FALSE)
+      ->addWhere('api_params', 'IS NOT NULL')
+      ->addChain('displays', \Civi\Api4\SearchDisplay::get()->addWhere('saved_search_id', '=', '$id'))
```

Comment:
@colemanw do we need to add a FALSE onto this APIv4 call here?

Reply:
I had to double-check it but no - chained api calls always inherit the `$checkPermissions` setting of the parent.

Reply:
Interesting, I would have presumed it was the other way around that that chain overwrote the parent but hey

Reply:
Unfortunately the api4 syntax doesn't really allow for that. `$checkPermissions` is either true or false, it can't be `undefined`. So there's (currently) no way to tell the difference between passing in a value vs accepting the default. Hence the parent api wouldn't know if the chained api call you passed in has an explicit value set for that param or not.